clang-tools-extra/docs/clang-tidy/checks/list.rst

```diff
@@ -275,6 +275,7 @@ Clang-Tidy Checks
    :doc:`modernize-raw-string-literal <modernize/raw-string-literal>`, "Yes"
    :doc:`modernize-redundant-void-arg <modernize/redundant-void-arg>`, "Yes"
    :doc:`modernize-replace-auto-ptr <modernize/replace-auto-ptr>`, "Yes"
+   :doc:`modernize-replace-memcpy-with-std-copy <modernize/replace-auto-ptr>`, "Yes"
```

Comment:
wrong link to check, didn't you use clang-tools-extra/clang-tidy/add_new_check.py to create a check ?

Reply:
And wrong check name